app/src/main/java/io/intrepid/contest/screens/contestcreation/addcategoriestocontest/AddCategoriesFragment.java

```diff
@@ -76,6 +76,11 @@ public void onNextClicked() {
     }
 
     @Override
+    public void onFocus() {
+
```

Comment:
if a method intentionally does nothing, add a comment about it being an intentional no-op so that its clear that this is intentional.